### PR TITLE
Add magic crayon

### PR DESCRIPTION
**Pull request by Leander-0**
## About the PR

As always thanks @GentleButter for the incredible sprites :^)

Adds magic crayon, basically a crayon with 999.999 charges so people that want to do large scale art dont need to buy 30 different packs of crayons. also included on the loadouts for the mere price of 30K.

## Why / Balance

Still can be eaten but will take 8 bites so you dont accidentally swallow a crayon the size of a phone if you missclick.

## How to test

Get to fun loadouts at the end select the magic crayon.
have 30K to spend.
paint the world red.

**Changelog**

:cl: Leander
- add: Added magic crayon and its selection on the Fun loadouts, paint the world with it!

### CODE REVIEW

**Comment by whatston3** [comment]
I put up a set of suggestions onto [whatston3:magic-crayon-suggestions](https://github.com/Leander-0/frontier-station-14/compare/Leander-0:magic-crayon...whatston3:magic-crayon-suggestions?expand=1).  I think the changes are reasonable, here's what I changed:

Nice things:
* No more "99700/99999" in the bar
* Better YAML parenting, smaller definition
* Animated in-hand sprites
* Gave attributions to other Paradise sprites missing a commit (mentioned the source DMI) - cleanup for things as I saw them, pardon the mess

Toss ups:
* Tastes "magical" (it's a magic crayon), not "sweet"
* Tweaked the description (it should taste magical :( )
* Renamed ID to CrayonMagic to match with CrayonRainbow, CrayonRed...

No fun allowed:
* Created a small animated version of the existing rainbow crayon instead of the new sprite, which, while nicely animated, is oddly large.
* Changed the crayon to be made of nothing, like plastic bananas (no incentives to eat it beyond it tasting like magic).

Small video showing how the crayons look in-game with a normal blue and rainbow crayon for comparison:

https://github.com/user-attachments/assets/d481d8a0-9e26-4370-be3b-458aa7003d1f

**Comment by Leander-0** [comment]
Oh wow not only you went above but also beyond, i will be pushing the changes but i do find it a shame that the other sprite was tossed out.

**Comment by whatston3** [comment]
> Oh wow not only you went above but also beyond, i will be pushing the changes but i do find it a shame that the other sprite was tossed out.

It's only an opinion, I'm just one guy - I do think it's a nice sprite, and could make for a neat fuel rod or some intermediate refined material, but it seems really big for a crayon.  Happy to revert if there's consensus the other way, and don't mean to step on toes.

**Review by whatston3 — APPROVED** [review]
Second set of eyes appreciated on this but I'm happy with it.

File: Resources/Prototypes/_NF/Entities/Objects/Fun/magic_crayon.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - type: FlavorProfile
+    flavors:
+    - chewy
+    - magical
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        reagents:
+        - ReagentId: Nothing
+          Quantity: 40
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Entities/Objects/Fun/magic_crayon.yml:28]: Real magic right there.

**Comment by dvir001** [comment]
Missing the option to yeet it and have it embed into someone but overall fine.

**Review by whatston3 — APPROVED** [review]

**Pull request merged**